frontend/src/app/testQueue/TestCardForm/TestCardForm.utils.tsx

```diff
@@ -22,11 +22,14 @@ import {
 } from "./types";
 
 /** Add more options as other disease AOEs are needed */
-export enum AOEFormOption {
-  COVID = "COVID",
-  HIV = "HIV",
-  NONE = "NONE",
-}
+export const AOEFormOption = {
+  COVID: "COVID",
+  HIV: "HIV",
+  SYPHILIS: "SYPHILIS",
+  NONE: "NONE",
+} as const;
+
+type AOEFormValues = (typeof AOEFormOption)[keyof typeof AOEFormOption];
```

Comment:
Can we keep `AOEFormOption` as an enum instead of changing it to use `as const`? I think having it as enum is helpful in that it restricts values from being interchangeable with strings. Plus I think it's more readable using the enum for the type directly rather than creating `AOEFormValues`